Neos.Neos/Tests/Behavior/Features/Bootstrap/RoutingTrait.php

```diff
@@ -168,7 +168,11 @@ abstract protected function getContentRepository(): ContentRepository;
      */
     public function theDocumenturipathProjectionIsUpToDate(): void
     {
-        $this->getContentRepository()->catchUpProjection(DocumentUriPathProjection::class);
+        if ($this->lastCommandOrEventResult === null) {
+            // we just blocked in "The graph projection is up to date"
+            return;
+        }
+        $this->lastCommandOrEventResult->block();
```

Comment:
dont we need to set it to null here as well?

(like in `theGraphProjectionIsFullyUpToDate`?)


also in general it seem to be better to use 

> The graph projection is fully up to date

instead no?

Reply:
@skurfuerst did you see that comment?

Reply:
Ah thanks. I can push a follow up